ocs_ci/ocs/ui/base_ui.py

```diff
@@ -195,6 +202,53 @@ def take_screenshot(self):
         self.driver.save_screenshot(filename)
         time.sleep(0.5)
 
+    def do_clear(self, locator, timeout=30):
+        """
+        Clear the existing text from UI
+
+        Args:
+            locator (tuple): (GUI element needs to operate on (str), type (By))
+            timeout (int): Looks for a web element until timeout (sec) occurs
+
+        """
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((locator[1], locator[0])))
+        element.clear()
+
+    def wait_until_expected_text_is_found(self, locator, expected_text, timeout=300):
+        """
+        Method to wait for a expected text to appear on the UI (use of explicit wait type),
+        this method is helpful in working with elements which appear on completion of certain action and
+        ignores all the listed exceptions for the given timeout.
+
+        Args:
+            locator (tuple): (GUI element needs to operate on (str), type (By))
+            expected_text (str): Text which needs to be searched on UI
+            timeout (int): Looks for a web element repeatedly until timeout (sec) occurs
+        return:
+            bool: Fetches the actual text and returns True if the expected text and actual text are same,
+                AssertionError otherwise.
+                TimeOutException if the element text is not found during the given time.
+
+
+        """
+        wait = WebDriverWait(
+            self.driver,
+            timeout=timeout,
+            poll_frequency=1,
+            ignored_exceptions=[
+                NoSuchElementException,
+                ElementNotVisibleException,
+                StaleElementReferenceException,
+                TimeoutException,
+            ],
+        )
+        wait.until(ec.visibility_of_element_located((locator[1], locator[0])))
+        text_found = (self.driver.find_element(locator[1], locator[0])).text
```

Comment:
Can be done with one command:
wait.until(ec.text_to_be_present_in_element((locator[1], locator[0]), expected_text))